packages/svgs/src/scripts/load-icons.ts

```diff
@@ -0,0 +1,93 @@
+import * as path from "path";
+import * as fs from "fs";
+import camelCase from "camelcase";
+import {
+    ICONS_SIZES 
+} from "./constants.ts";
+import { getAllFiles } from "./helper.ts";
+
+interface DataItem {
```

Comment:
We'll have to find a better name for this, i can't know what this is. I'll check to code and suggest a new name soon

Reply:
interface IconMetadata {
    filePath: string;   <-- fill this property from handleName instead, and rename the method to handleMetadata
    content: string;
    group: number;
}

export interface Icon extends IconMetadata {
    name: string;
}

Reply:
I'm a bit confused because DataItem, now IconMetadata has name and size(group), why are you changing the props?

Reply:
should content be filled in handleMetadata as well then?

Reply:
you also removed name from the matadata?